_posts/necessities/2022-12-28-tempeh.md

```diff
@@ -0,0 +1,62 @@
+* Sandor Katz's [_The Art of Fermentation_][katz]
+* [_The Book of Tempeh_][shurtleff]
+* [_Miso, Tempeh, Natto & Other Tasty Ferments_][shockey]
+
+I'd like to experiment next year with making non-soy bean tempehs (chana daal???) as well as using banana leaves which are abundant around the neighborhood. 
+
+**Ingredients and Tools**
+* soy beans
+* quart sized zip lock bag 
+* toothpick 
+* electric proofer, or an ambient air temperature of 85-91 degrees fahrenheit for at least 12 hours
+* vinegar (basic ass white vinegar is fine and preferred here)
+* tempeh culture
+
+**Directions**
+
+* Purchase a tempeh starter culture. I got mine from [Preserved][preserved] in Oakland. 
+* Soak soy beans for at least 24 hours. 2 cups of dry soy beans makes roughly 2 sandwich bags of tempeh. According to Shurtleff and Katz, soaking longer is traditional to provide some acidification as protection from bad bacteria growth. There's also been some [studies](https://pubmed.ncbi.nlm.nih.gov/1905283/) on this recently.
+* "Husk" soy beans from their shells. This is incredibly tedious and if I ever want to scale up, I want to machine this step. You can sort of rub the husks off while the soy beans are in water and then let the husks float. Then remove the husks gently with a sieve. The idea behind this step is to allow the tempeh culture to permeate the bean -- the husks are too waxy for the culture to let the beans stick together. I didn't do a *great* job every time I dehusked, and when that happens you end up with a tempeh that is a little more crumbly and less like a block (still delicious though). 
+* Steam husked soy beans for ~30-45 minutes. 
+* Completely dry soy beans by placing them in a single layer on sheet pans and drying with a paper towel. You can also blow dry with a blow dryer apparently.
+* Add tempeh culture, in accordance with the directions on the tempeh culture packet and ~1 tablespoon of vinegar for every 2 cups of dry soy beans. This is to add additional acifidifcation to prevent bad bacteria growth.
+* Poke holes every 1/2 inch into quart sized ziplock/sandwich bags with a toothpick. 
+* Pour tempeh into zip block bags, make sure they're extremely flat. Place in proofer at 88 degrees fahreneheit. After 12 hours, check on tempeh. If it's more solid feeling and you start to see white stuff develop, turn the proofer off and wait an additional 12 hours for it to fully ferment. Then transfer to fridge to slow fermentation. It is VERY ACTIVE! and will continue to mature in the fridge even. Store bought tempeh is pasteurized.
+
+**Some tempeh strategies**
+
+We have two high level strategies for making tempeh--
+1. "Fry" and then braise in sauce.
+2. Marinade in sauce and then simmer. Tempeh can hang out in marinade in the fridge for a day or two, and it's a great way to slow the maturing step if you have too much homemade tempeh on hand. The tempeh if it's not *very* firm can potentially fall apart in the braising though.
+
```

Comment:
You should also add blanching in boiling water: this is for reducing any bitterness from over fermenting. It can also be followed by a braise in a sauce (which means that you've avoided frying it).